docker-compose.override.yml

```diff
@@ -18,7 +18,7 @@ services:
       - db
   backend:
     image: akvo/akvo-clojure-lein:20210124.114043.4437caf
-    command: ["lein", "repl", ":headless"]
+    command: ["tail", "-f", "/dev/null"]
```

Comment:
IMO these settings are only useful in your env, so please remove them from this PR

Reply:
Removed and replaced with newer values https://github.com/akvo/unep-gpml/commit/2843790c7ed570ec976ab5e3f9e92eb65779b6df#diff-2286cdeca85adb587a9640aa34352ba7c2fb6e49c0129dab05892ff67d0d4c58R21